docs/vm-ee-guide.md

```diff
@@ -0,0 +1,267 @@
+# Onboard
```

Comment:
Should this include how VMs are onboarded and offboarded?

Reply:
I am not sure about offboard, can't think what we can add under offboard. Onboard is already in the doc.